.github/workflows/deploy-dev.yml

```diff
@@ -54,6 +54,8 @@ jobs:
           # Pull ranges from shared LAA IP ranges and then remove spaces,
           # replace linebreaks with commas, remove last comma, and escape commas for helm input
           SHARED_IP_RANGES_LAA=$(curl -s https://raw.githubusercontent.com/ministryofjustice/laa-ip-allowlist/main/cidrs.txt | tr -d ' ' | tr '\n' ',' | sed 's/,/\\,/g' | sed 's/\\,$//')
+          PINGDOM_IP_RANGES=$(curl -s  https://my.pingdom.com/probes/ipv4 | tr -d ' ' | tr '\n' ',' | sed 's/,/\\,/g' | sed 's/\\,$//')
+          echo "PINGDOM_IP_RANGES=$PINGDOM_IP_RANGES" >> $GITHUB_ENV
```

Comment:
Can we consistently call Pingdom IPs, `PINGDOM_IPS`. As they are not ranges, rather they are singular IP addresses.